lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php

```diff
@@ -274,4 +269,17 @@ private function getJoinTableName($assoc, $class, $platform)
 
         return $this->em->getConfiguration()->getQuoteStrategy()->getJoinTableName($assoc, $class, $platform);
     }
+
+    /**
+     * @param string              $tableName
+     * @param AbstractPlatform    $platform
+     *
+     * @return string
+     */
```

Comment:
Please remove this entire docblock, it's redundant.

Reply:
Type hints over phpdoc, got it; will tweak now